### ISSUE TITLE
Corpse Killer: Enemies do not behave as they should.

### ISSUE THREAD

**Issue by Silanda** [libretro/opera-libretro#198]
In short, multiple types of zombies are supposed to attack you, either by walking towards you from varying positions on the screen, or by moving inwards from the sides of the screen. This doesn't happen using Opera. Instead, large groups of the same type of enemy are spawned in the same position walking towards you. No enemies attack from the side, and there doesn't seem to be any randomisation. Unfortunately, the recent RNG changes haven't fixed this.

Incidentally, the only 3DO emulator where this game seems to work more or less correctly is the old 3DOplay. IIRC it behaves slightly differently, but the game doesn't work properly on Phoenix either.

**Comment by trapexit** [comment]
I believe I figured it out. I don't know the absolute root cause in the sense of why the emulator was written as it was vs the hardware behavior (or what that hardware behavior is) but I was able to get it working as expected. Though I need to do some more testing. Should have something this week.

![image](https://github.com/libretro/opera-libretro/assets/197676/2e0bbeb7-7633-47b7-9b24-ced35a6638f3)
![image](https://github.com/libretro/opera-libretro/assets/197676/292853f3-f029-4ec5-9060-c1dd99ff9339)
